autocertkit/utils.py

```diff
@@ -93,6 +93,7 @@ def configure_logging():
 
 
 configure_logging()
+set_logger(log)
```

Comment:
```suggestion
```
Hi, just a suggestion: Could this be moved inside of __main__ in order to allow importing this file for unit tests without the import itself reconfiguring logging?
```py 
if __name__ == '__main__':
```
(inside the check at line 2049): https://github.com/xenserver/auto-cert-kit/pull/206/files#diff-be598b7cf05301424c37a42e8c74ba5b85ccbb5302c6dc09cb1d2b0dc89661cfR2049

Reply:
Yes, it's able to.
ACK has 2 `main` entrances, we put logger initialization to utils.py is to simply use, means developer needs not to explicitly initialize logger, including in UT. And that loses flexibility indeed.
So won't change currently.